.agent-os/instructions/core/post-execution-tasks.md

```
@@ -0,0 +1,249 @@
+</content_requirements>
+
+</step>
+
+<step number="7" subagent="project-manager" name="completion_summary">
+
+### Step 7: Completion Summary
+
+Use the project-manager subagent to create a structured summary message with emojis showing what was done, any issues, testing instructions, and PR link.
+
+<summary_template>
+  ## ✅ What's been done
```

markdownlint-cli2 (0.17.2)

Headings must start at the beginning of the line

(MD023, heading-start-left)